packages/components/src/slot-fill/bubbles-virtually/use-slot.ts

```diff
@@ -21,8 +21,10 @@ export default function useSlot( name: SlotKey ) {
 
 	const api = useMemo(
 		() => ( {
-			updateSlot: ( fillProps: FillProps ) =>
-				registry.updateSlot( name, fillProps ),
+			updateSlot: (
```

Comment:
Where is this one used. or is it not used at all? I see the function arguments modified but I don't see where the calling of the function is updated.

Reply:
The `updateSlot` method is used in the `Slot` component. Called on every render, in a effect without dependencies, to make sure that the latest `fillProps` is always synced promptly to the registry.

The bug being fixed is triggered when, for any reason, the `Slot` is rerendered one more time before unmounting. Then `updateSlot` would store a stale `fillProps` value at a time when a new `Slot` is already being mounted. The `unregisterSlot` method already has exactly the same `ref` check.

Reply:
But this one updateSlot that is returned from useSlot, is not used right?

Reply:
Oh yes, this is a good observation. The `updateSlot` and `unregisterSlot` functions returned from `useSlot` are not really used. And I think I could get rid of the entire `api` object and return only the `slot`. I'll do that in a followup PR.